app/views/discussions/print.html.haml

```diff
@@ -0,0 +1,101 @@
+.print-container
+  .row.title
+    .col-md-9
+      %h1= @discussion.title
+      .groups-title
+        - if @group.is_subgroup?
+          %h3= @group.parent.name
+          = content_tag :span, "\u25B6", class: 'name-separator'
+        %h3=@group.name
+  .row.context
+    .col-md-9
+      = @discussion.description
+    .col-md-2.col-md-offset-1
+      .discussion-details
+        = t("discussion_context.started_print",
+            when: time_ago_in_words(@discussion.created_at),
+            who: @discussion.author.name)
+  - if @motions
+    .row
+      .col-md-12
+        %h2= t :proposals
+      - @motions.each do |motion|
+        .col-md-9
+          - if motion == @discussion.current_motion
+            %h3= t :current_decision
+          %h3= motion.title
+          .proposal-description
+            = motion.description
+          .proposal-details
+            %ul
+              - if motion.voting?
+                %li= t(:closing) + ": " + motion.closing_at.strftime("%d %B, %I.%M %p %Z")
+              - else
+                %li= t(:closed_at) + ": " + motion.closed_at.strftime("%d %B, %I.%M %p %Z")
+              %li= t :proposed_by_who_on, when: motion.created_at.strftime("%d %B, %I.%M %p %Z"), who: motion.author_name
+            - if motion.has_outcome?
+              %strong= t(:outcome) + ": "
+              %span
+                = motion.outcome
+        .col-md-9
+          .motion-graph.clearfix
+            %canvas.pie{id: "graph_#{motion.id}", height: 150, width: 150}
+            .legend
+              .item.yes
+                = t(:'position_verbs.yes')
+                (#{motion.yes_votes_count})
+              .item.abstain
+                = t(:'position_verbs.abstain')
+                (#{motion.abstain_votes_count})
+              .item.no
+                = t(:'position_verbs.no')
+                (#{motion.no_votes_count})
+              .item.block
+                = t(:'position_verbs.block')
+                (#{motion.block_votes_count})
+
+              - if motion.closed?
+                = t :engagement_on_closed_proposal,
+                    percent: motion.percent_voted,
+                    numerator: (motion.group_size_when_voting - motion.members_not_voted_count),
+                    denominator:  motion.group_size_when_voting
+              - else
+                = t :engagement_on_open_proposal,
+                    percent: motion.percent_voted,
+                    numerator: (motion.group_size_when_voting - motion.members_not_voted_count),
+                    denominator:  motion.group_size_when_voting
+            = render('motions/motion_pie_javascript', motion: motion)
+        .col-md-9
+          %h3= t :positions
+          %ul.positions
+            - if motion.has_votes?
+              - motion.grouped_unique_votes.each do |vote|
+                - icon = display_proposal_activity_icons_for_print(vote)
```

Comment:
Don't think we have to declare this as a variable, since we're just using it once.

```
image_tag proposal_activity_icon_for(vote)
```